.github/actions/spelling/Spelling.md

```diff
@@ -0,0 +1,14 @@
+# Spell checking
+
+## Basics
+
+Since this action is an extension of [check-spelling](https://github.com/check-spelling/check-spelling), familiarize yourself with its concepts and configuration options, especially about [dictionary files](https://github.com/check-spelling/check-spelling).
+One important concept is that the spell checking process replaces matched words with a space character and built-in dictionaries are checked first. The order of entries within a dictionary file also matters.
```

Comment:
```suggestion
Since this action is an extension of [check-spelling](https://github.com/check-spelling/check-spelling), familiarize yourself with its concepts and configuration options, especially about [dictionary files](https://github.com/check-spelling/check-spelling).

One important concept is that the spell checking process replaces matched words with a space character and built-in dictionaries are checked first. The order of entries within a dictionary file also matters.
```